programs/cli_wallet/main.cpp

```diff
@@ -116,28 +116,27 @@ int main( int argc, char** argv )
          return 0;
       }
 
+      // set up logging
       fc::path data_dir;
       fc::logging_config cfg;
       fc::path log_dir = data_dir / "logs";
-
       fc::file_appender::config ac;
-      ac.filename             = log_dir / "rpc" / "rpc.log";
+      ac.filename             = log_dir / "cli_rpc" / "rpc.log";
```

Comment:
It works but i think we should think a bit how we want it to be. 

In the `cli_wallet` the `data_dir` cant be selected from the command line so `data_dir` is always current path. 

This will result always in a `logs/cli_rpc/rpc.log` structure. I understand the rename makes the log clear but conflict is unlikely as witness node will store logs by default in `witness_node_data_dir/logs/rpc`.

Maybe we should force `data_dir` to be `cli_wallet_data_dir`(can also add the option to command line) and drop the rpc logs and wallet.json in there.

Reply:
perhaps a directory named "cli_wallet_logs".

Reply:
Made directory cli_wallet_logs